tests/unit/test_charm.py

```diff
@@ -168,6 +170,81 @@ def test_given_unit_is_leader_and_generate_self_signed_certificate_set_to_true_b
             self.harness.charm.unit.status,
         )
 
+    def test_given_unit_is_leader_and_generate_self_signed_certificate_set_to_true_default_validity(  # noqa: E501
+        self,
+    ):
+        self.harness.set_leader(True)
+        self.harness.add_relation("replicas", self.harness.charm.app.name)
+
+        self.harness.update_config(
+            key_values={"generate-self-signed-certificates": "true", "ca-common-name": "RootCA"}
+        )
+
+        rel = self.harness.charm.model.get_relation("replicas")
+
+        assert isinstance(rel, Relation)
```

Comment:
Those 2 asserts aren't necessary in the context of the test. The same is applicable elsewhere here.

Reply:
Actually, I placed them as otherwise mypy would be complaining (because they might also be None, in theory, e.g. rel is typed as Optional[Relation]). I suppose there is nothing wrong to have them in.